test/integration/test_connection_winrm.inventory

```diff
@@ -0,0 +1,10 @@
+[winrm]
```

Comment:
This should probably just be combined with inventory.winrm.template (with the vagrant-ish details scrubbed out), since anyone wanting to run this will need to adjust accordingly (just as with standard winrm test inventory).